src/views/dashboard/index.vue

```diff
@@ -155,16 +155,22 @@ export default {
     })
 
     this.axios.get(this.services['oms-events'] + '/mine/participating').then((response) => {
-      for (const event of response.data.data) {
-        if (moment().isSameOrBefore(event.starts)) {
-          this.events.future.push(event)
-        } else {
-          this.events.past.push(event)
+      this.axios.get(this.services['oms-statutory'] + '/events/mine').then((response2) => {
```

Comment:
Also, while we are here, can you also rename this to "statutoryResponse" and the one above to "eventsResponse"? That will make it more clear.